plugins/casutg/src/main/java/com/blazemeter/jmeter/threads/arrivals/ArrivalsThreadGroup.java

```diff
@@ -38,6 +38,11 @@ public void start(int groupIndex, ListenerNotifier listenerNotifier, ListedHashT
         }
     }
 
+    @Override
```

Comment:
All of these overrides with `null` result look strange. Can you shortly explain why is it like that?

Reply:
addNewThread exist in the Abstract of ThreadGroup but never was used (is not called by any class). It's like a "garbage".
https://github.com/search?q=repo%3Aapache%2Fjmeter%20addNewThread&type=code

OpenModelThreadGroup by example add the following comment "Will not be implemented as the semantics of the API is unclear"
I'll add a comment similar to the one in OpenModel to make it clear why.

Reply:
Maybe implement it in AbstractDynamicThreadGroup, so the inheritors don't need to bother?